tools/ci_build/github/windows/extract_nuget_files.ps1

```diff
@@ -52,10 +55,22 @@ elseif ($aars.Count -gt 1) {
   Write-Error "Expected at most one Android .aar file but got: [$aars]"
 }
 
-
 New-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo -ItemType directory
 
-Copy-Item -Path $nuget_artifacts_dir\onnxruntime-win-x64-*\lib\* -Destination $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo
+# Check whether this is a training pipeline
+$is_training_pipeline = $false
+if (Test-Path -Path $nuget_artifacts_dir\onnxruntime-training-win-x64-*) {
+  $is_training_pipeline = $true
+  Write-Output "onnxruntime-training-win-x64-* dir exists. This is a training pipeline."
+}
+
+if ($is_training_pipeline) {
+  Copy-Item -Path $nuget_artifacts_dir\onnxruntime-training-win-x64-*\lib\* -Destination $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo
+}
+else {
+  Copy-Item -Path $nuget_artifacts_dir\onnxruntime-win-x64-*\lib\* -Destination $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo
```

Comment:
nit: would be good to capture in a comment why we do these copies now that you've spent time figuring it out